cilium-cli/connectivity/builder/manifests/allow-all-except-world.yaml

```diff
@@ -15,17 +15,17 @@ spec:
   - toEndpoints:
     - matchExpressions:
       - { key: io.cilium.k8s.policy.cluster, operator: Exists }
-  # When node-local-dns is deployed with local IP,
-  # Cilium labels its ip as world.
-  # This change prevents failing the connectivity
-  # test for such environments.
+  # For nodelocaldns using a link-local address (the "localip" mode).
   - toPorts:
     - ports:
       - port: "53"
         protocol: UDP
       - port: "53"
         protocol: TCP
+    # Including 'world' here is a legacy workaround for pre-MR#25298 behaviour,
```

Comment:
I didn't follow the discussions - but looking at PR 25298 it was merged in 1.13. We no longer support that version of Cilium in our test suite. Does that mean we can remove the `world` entry?

Reply:
> Should I change the comment to something like "Including 'world' here is believed to be a workaround for previous behaviour"

Yes, please add such a comment, referring to at least Cilium v1.16 as still needing it (even if the cause is unclear). The fact that the tests pass for you with Cilium v1.16 and a recent version of the CLI means that traffic can still be labelled with `world`. This means that once v1.16 is EOL we can re-consider removing it.